catalog/opensearch/catalog-opensearch-source/src/main/java/ddf/catalog/source/opensearch/impl/OpenSearchSource.java

```diff
@@ -390,6 +410,12 @@ private InputStream performRequest(WebClient client) throws UnsupportedQueryExce
     return stream;
   }
 
+  /** Package-private so that tests may set the foreign markup consumer. */
+  void setForeignMarkupBiConsumer(
```

Comment:
❓ Do we need this since we already have a constructor that takes in the `BiConsumer`?
✏️  We could replace the comment with Guava's `@VisibleForTest`